packages/web-components/src/toolbar/toolbar.spec.md

```diff
@@ -0,0 +1,56 @@
+# Toolbar
+
+## Component Description
+
+A Toolbar is a container for grouping a set of controls: action controls (e.g. button, icon button), input fields, dropdown, and menu button. Each control will be a web component.
+
+Checkbox too?
+
+The actions and input fields on a toolbar can include dropdowns.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/QfHyx6ApwFEcw5q4pM9CSZ/Toolbar?node-id=2351-6672&t=MuajdFxoCBjwlK4w-0)
+
+[Link to Fluent 2 Spec](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-toolbar--default)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr appearance: "static" | "contextual"
```

Comment:
I see this in the design file, but not in the Fluent React control. You might consider reaching out to developer of the Fluent React control to see why they omitted this or if they plan to add it later. @miroslavstastny do you have any insights here?

Reply:
I plan to remove this attribute. I had a good chat with the engineer who built the react-toolbar. The appearance is about its usage and not a property of the component itself.